conf/application.conf

```diff
@@ -155,6 +155,11 @@ toggles {
 
 urls {
 
+  tar{
+    host = "http://localhost:9280"
+    url = "/account"
+  }
```

Comment:
This should go inside the services section and be called tax-account-router-frontend to match app-config-base. app-config-base will need to be updated to match